docs/plugins.jetbrains.com/getting_started.html

```diff
@@ -12,7 +12,6 @@
   </li>
   <li>From the left side plugin's toolbar select <b>Open Machete File</b></li>
   <li>Edit the file (.git/machete) if needed to reflect the actual branch layout; the completion (Ctrl + Space) may be useful</li>
-  <li>Refresh the graph: save file and click <b>Refresh Status</b> (Ctrl + F5, or Cmd + R on macOS) button from the left side plugin's toolbar</li>
```

Comment:
Pls update in the plugin website as well

Reply:
done